### PR TITLE
Update default automatic MetadataRefresh to 1 day.

### PR DESCRIPTION
**Pull request by polita**
For #401. @brentschmaltz , please take a look.

### CODE REVIEW

**Comment by brentschmaltz** [comment]
:shipit:

File: src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs [hunk 1]
```diff
@@ -45,7 +45,7 @@ public class ConfigurationManager<T> : IConfigurationManager<T> where T : class
         /// <summary>
         /// 5 days is the default time interval that afterwards, <see cref="GetConfigurationAsync()"/> will obtain new configuration.
```
Comment (PinpointTownes) [inline src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs]: Nice, the XML documentation is now incorrect... maybe it was worth updating it? :sweat_smile:

Comment (polita) [inline src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs]: Thanks, I'll fix it.

Comment (polita) [inline src/Microsoft.IdentityModel.Protocols/Configuration/ConfigurationManager.cs]: See PR #410

**Pull request merged**